docs/general/dns-providers.md

```diff
@@ -333,6 +333,27 @@ EDNS Client-Subnet is a method that includes components of end-user IP address d
 | DNS, IPv4      | `84.200.69.80` and `84.200.70.40`                  | [Add to AdGuard](sdns://AAAAAAAAAAAADDg0LjIwMC42OS44MA) |
 | DNS, IPv6      | `2001:1608:10:25::1c04:b12f` and `2001:1608:10:25::9249:d69b`            | [Add to AdGuard](sdns://AAAAAAAAAAAAHFsyMDAxOjE2MDg6MTA6MjU6OjFjMDQ6YjEyZl0) |
 
+[FutureDNS](https://futuredns.me/) is a privacy friendly DNS service that blocks ads, trackers, malware and that does not log any data.
+
+|Anycast Network | Address                    |                                                        |
+|----------------|------------------------------------------------------------------|------------------|
+| DNS-over-HTTPS | `https://dns.futuredns.me` | [Add to AdGuard](sdns://AgcAAAAAAAAAAAAQZG5zLmZ1dHVyZWRucy5tZQA)
+| DNS-over-TLS   | `tls://dns.futuredns.me` | [Add to AdGuard](sdns://AwcAAAAAAAAAAAAQZG5zLmZ1dHVyZWRucy5tZQ)
+| DNS-over-QUIC  | `quic://dns.futuredns.me` | [Add to AdGuard](sdns://BAcAAAAAAAAAAAAQZG5zLmZ1dHVyZWRucy5tZQ)
```

Comment:
Cannot verify that this one works:
```
% dnslookup example.org quic://dns.futuredns.me    
dnslookup v. 1.7.0
2022/07/05 19:48:45 Cannot make the DNS request: opening quic connection to quic://dns.futuredns.me:853: timeout: no recent network activity
```

```
% dnslookup example.org sdns://BAcAAAAAAAAAAAAQZG5zLmZ1dHVyZWRucy5tZQ
dnslookup v. 1.7.0
2022/07/05 19:49:05 Cannot make the DNS request: opening quic connection to quic://dns.futuredns.me:853: timeout: no recent network activity

```

Please use dnslookup or a similar tool to verify that the addresses (and DNS stamps) are working:
https://github.com/ameshkov/dnslookup

Could it be that the resolver uses a draft version of DoQ?

Reply:
Ah, it uses AdGuard Home according to the description.

@ainar-g have we already published the final DoQ version to the stable branch?

Reply:
problem solved
i didn't realize that 853/udp ports were closed
now i tested and everything is working correctly